cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -605,7 +605,7 @@ parameter_types! {
 }
 
 parameter_types! {
-	pub TreasuryAccount: AccountId = PalletId(*b"py/trsry").into_account_truncating();
+	pub TreasuryAccount: AccountId = TREASURY_PALLET_ID.into_account_truncating();
```

Comment:
Re-use the TREASURY_PALLET_ID for now. TREASURY_PALLET_ID defined to be same thing anyway:
```
PalletId(*b"py/trsry")
```

Reply:
Sounds fine. The local fees can all go to the common treasury account for BridgeHub.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/999/commits/5c03ad858d2f4c046b7dc26ce9cd6c1e5d615a34.